ChaRo-iOS/ChaRo-iOS/Source/Views/VCs/CreatePostScene/MapScene/AddressConfirmVC.swift

```diff
@@ -20,45 +21,33 @@ class AddressConfirmVC: UIViewController {
     public var presentingCellIndex = -1
     private var isFirstLoaded = true
     
-    private var backButton: UIButton = {
-        let button = UIButton()
-        button.setBackgroundImage(ImageLiterals.icCircleBack, for: .normal)
-        button.addTarget(self, action: #selector(popCurrentView), for: .touchUpInside)
-        return button
-    }()
-    
+    private var backButton = UIButton().then{
```

Comment:
```suggestion
    private var backButton = UIButton().then {
```
요기 커밋에서도 then 뒤에 스페이스 확인부탁해용

Reply:
반영했습니다 :) https://github.com/TeamChaRo/ChaRo-iOS/pull/192/commits/879a94e089f395af987f7355296c71fd3219dfa4